resources/assets/projectred/lang/fr_FR.lang

```diff
@@ -1,20 +1,20 @@
 itemGroup.projectred.core=Project Red: Base
 
-item.projectred.core.itemResource.partplate.name=Plaque pour Circuit
+item.projectred.core.itemResource.partplate.name=Plaque de Circuit
 item.projectred.core.itemResource.partconductiveplate.name=Plaque Conductive
 item.projectred.core.itemResource.partwiredplate.name=Plaque Cablée
 item.projectred.core.itemResource.partbundledplate.name=Plaque Câblée avec des Câbles Empaquetés
 item.projectred.core.itemResource.partanode.name=Anode
 item.projectred.core.itemResource.partcathode.name=Cathode
 item.projectred.core.itemResource.partpointer.name=Pointeur
-item.projectred.core.itemResource.partsiliconchip.name=Puce de Silicium
-item.projectred.core.itemResource.partenergizedsiliconchip.name=Puce de Silicium Énergisée
+item.projectred.core.itemResource.partsiliconchip.name=Puce de Silicone
+item.projectred.core.itemResource.partenergizedsiliconchip.name=Puce de Silicone Énergisée
```

Comment:
Silicone (English: silicone) and Silicium (English: Silicon) are not the same thing. And in this case, I think Silicium (silicon) is the correct term.

Reply:
yes, i think that Silicon is the best for this, keeping it